lib/acme-client.rb

```diff
@@ -1,3 +1,7 @@
+# This file lives here for backward compatibility only and should be deprecated soon
+# gem named acme-client should be required as
```

Comment:
TIL, thanks. I still think it can't harm and it seems to be common practice to offer both.